### DISCUSSION TITLE
歌词api

### DISCUSSION THREAD

**Discussion by yangwenjie1231** [Beadd/MusicDownloader#68] [Ideas]
这是网易云音乐的歌词api，可以整合歌词下载，api：http://music.163.com/api/song/lyric?os=pc&id=歌曲id&lv=-1&kv=-1&tv=-1

**Comment by Beadd** [comment]
> 这是网易云音乐的歌词api，可以整合歌词下载，api：[http://music.163.com/api/song/lyric?os=pc&id=歌曲id&lv=-1&kv=-1&tv=-1](http://music.163.com/api/song/lyric?os=pc&id=%E6%AD%8C%E6%9B%B2id&lv=-1&kv=-1&tv=-1)

歌词已经自动嵌入到音乐文件里了

**Reply by yangwenjie1231** [reply]
但是一些MP3随身听只能识别lrc格式的，可能对于这部分用户不太友好

**Reply by Beadd** [reply]
> 但是一些MP3随身听只能识别lrc格式的，可能对于这部分用户不太友好

好的，新版本会同时下载歌词并添加一个下载歌词的开关

**Reply by yangwenjie1231** [reply]
给个star

**Reply by yangwenjie1231** [reply]
请问现在可以下载lrc歌词了吗？

**Reply by Beadd** [reply]
最近事情比较多，我争取这几天就重构了

**Comment by Beadd** [comment]
已更新，[v5.0.0](https://github.com/Beadd/Creamplayer/releases/tag/v5.0.0)
